requirements.txt

```diff
@@ -1,8 +1,8 @@
 descartes
-geopandas
+geopandas>=1.0.0
```

Comment:
Sorry to be pain but this may actually cause other conflicts :D and I don't think you need a pin here at all.

Reply:
Ok, I will remove this pin ;)